### PR TITLE
Maestro Cloud -> Robin

### PR DESCRIPTION
**Pull request by Fishbowler**
* Lots more Robin, with a few Maestro Cloud caveats
* Added a canonical list of inputs at the top
* Fixed some formatting that the linter was complaining about

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,42 +59,47 @@ on:
...
+For more information on triggering workflows, check out [GitHub's documentation](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows).
+
+## Android
 
 ```yaml
 - uses: mobile-dev-inc/action-maestro-cloud@v1.9.4
   with:
-    api-key: ${{ secrets.MAESTRO_CLOUD_API_KEY }}
+    api-key: ${{ secrets.ROBIN_API_KEY }}
+    project-id: 'proj_01abc3def456ghij78klmnopq9'
```
Comment (axelniklasson) [inline README.md:77]: @Fishbowler I guess pasting in the project id here wasn't intended? It's not secret, so we could in theory have real project id here but I propose we replace it with something like `<your_project_id>` 🙂

Comment (Fishbowler) [reply README.md:77]: It's a fake project ID, and our docs make it clear it's not a secret, so I wanted to include an example of the string format.

Chances are, folk will include the action once in a project - they're not going to abstract the value to a GH secret.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,43 @@
...
+## Using the action
 
 Add the following to your workflow. Note that you can use the `v1` tag if you want to keep using the latest version of the action, which will automatically resolve to all `v1.minor.patch` versions as they get published.
 
 ```yaml
 - uses: mobile-dev-inc/action-maestro-cloud@v1.9.4
   with:
-    api-key: ${{ secrets.MAESTRO_CLOUD_API_KEY }}
+    api-key: ${{ secrets.ROBIN_API_KEY }}
+    project-id: 'proj_01abc3def456ghij78klmnopq9'
```
Comment (axelniklasson) [inline README.md:13]:
```suggestion
    project-id: 'proj_01abc3def456ghij78klmnopq9' # replace this with your actual project id
```

Comment (Fishbowler) [reply README.md:13]: What about this in the first instance, then just the example ID in the remainder?
You're right in that my existing example takes a little parsing to notice that it's not a real ID.

```suggestion
    project-id: 'proj_01example0example1example2' # replace this with your actual project id
```

Comment (axelniklasson) [reply README.md:13]: That sounds great!

**Pull request merged**